### ISSUE TITLE
Labels for R version, CRAN mirror and shiny version

### ISSUE THREAD

**Issue by vvolodko** [rocker-org/shiny#64]
Often I need to check some info of Docker image before pulling it from DockerHub. There is API and tools for that, see https://github.com/containers/skopeo and https://hackernoon.com/inspecting-docker-images-without-pulling-them-4de53d34a604 .

It would be nice to have LABELs for versions of packages the `rocker/shiny` images (especially `:devel` and `:latest`) were built with.

The most interesting labels for me are:
* R version
* URL of CRAN stored in `/usr/local/lib/R/etc/Renviron`
* URL of MRAN snapshot stored in `/etc/environment`
* version of `shiny` package.
Perhaps the first three should be set in the `rocker/r-ver` Dockerfile.

If it sounds reasonable, I could prepare a patch.

**Comment by cboettig** [comment]
Yup, having these as `LABEL`s in the Dockerfile seems reasonable, given that the API makes it easier to inspect those than env vars.  I agree the first three would be more natural in `rocker/r-ver`

**Comment by eitsupi** [comment]
I think this has been resolved in the latest image, so close it.
